SamplesDashboard/SamplesDashboard/ClientApp/src/views/details/Details.tsx

```diff
@@ -88,27 +92,35 @@ export default class Details extends React.Component<any, any> {
         // call the statistics function
         this.statusStatistics(this.allItems);
         this.setState({
-            items: this.allItems,
+            items: this.allItems.sort((a, b) => (a.packageName.toLowerCase() >
+                b.packageName.toLowerCase()) ? 1 : -1),
```

Comment:
The sorting function can be abstracted so that it is easily readable and reusable. So that now items looks like
`items: sortItems(allItems, packageName.toLowercase, 'DESC)`,

Reply:
Oh yes, good catch!